packages/ui/src/primitives/alert.tsx

```diff
@@ -1,29 +1,61 @@
-import { cx } from 'cva';
 import * as React from 'react';
 
+import {
+  useAppearance,
+  mergeDescriptors,
+  type ParsedElementsFragment,
+  type ParsedDescriptor,
+} from '~/contexts/AppearanceContext';
 import * as Icon from './icon';
 
+export const layoutStyle = {
+  alert: {
+    className: 'border px-4 py-3',
+  },
+  alert__warning: {},
```

Comment:
These empty objects were necessary when `buildTheme` wanted to have every descriptor. With that restriction removed we wouldn't need these. We would need some runtime-validation though because otherwise `parsedElements` would return `undefined` if you tried to access `elements.alert__warning` with just the `layoutTheme`.

Reply:
I think we can either rely on `useAppearance()` to provide defaults, or let `mergeDescriptors()` gracefully handle `undefined`